server/src/views/frame.js

```diff
@@ -222,6 +222,7 @@ class Frame extends React.Component {
 
   clickFullPageButton(e) {
     e.preventDefault();
+    history.pushState({state: "frame"}, "", "#frame");
```

Comment:
Note that it the hash isn't useful unless when I open the URL and it includes `#frame` then I also navigate correctly to the full page view.